docs/source/support/index.rst

```diff
@@ -0,0 +1,12 @@
+==========================================================
+Support
+==========================================================
+
+.. toctree::
+   :titlesonly:
```

Comment:
Doing it this way makes the padding between the bullets smaller compared to other submenus, it looks a bit off.

Reply:
i didn't notice `:titlesonly` option affects the padding, can you elaborate?

Reply:
It's not about titlesonly but having the generated index compared to the hidden tree and list of docs as in other subindexes we have, it's just inconsistent.

Reply:
ok, seems like the point being discussed in https://github.com/scylladb/scylla-operator/pull/1504#discussion_r1371287253